File: src/Generated20260116.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260116 {
public:
    // Generated class 20260116_063038_801102
    int value = 491;
    string created = "20260116_063038_801102";
    
    int process() {
        int result = value;
        for (int i = 0; i < 8; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260116 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260116.go
package main

import "fmt"

type Generated20260116 struct {
    // Generated struct 20260116_063655_604018
    Value int
    Created string
}

func (c *Generated20260116) Process() int {
    result := c.Value
    for i := 0; i < 5; i++ {
        result += i * 2
    }
    return result
}

func main() {
    obj := Generated20260116{Value: 433, Created: "20260116_063655_604018"}
    fmt.Println(obj.Process())
}